src/persistence.js

```diff
@@ -39,6 +39,9 @@ export default function _Persistence(mpInstance) {
             if (!mpInstance._Store.isLocalStorageAvailable) {
                 mpInstance._Store.SDKConfig.useCookieStorage = true;
             }
+            if (mpInstance._Store.getPrivacyFlag('SDKState')) {
```

Comment:
I would consider moving this to the top of the function so that we don't read from cookies/localStorage at all if privacy is turned on.